### DISCUSSION TITLE
完啦，非酋头子

### DISCUSSION THREAD

**Discussion by fufumo** [share121/inter-knot#3047] [其他]
![PixPin_2024-08-19_14-28-19](https://github.com/user-attachments/assets/df900d3d-c757-44cd-9ee1-42c0cefd6fb3)

**Comment by Nie7bai** [comment]
这还非？？？楼主是来炫耀的吗<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/33b49a8bbead2a9e85290a75160d584e_2942391851204099012.png" alt="刃 来了" width="75" >
